content/learning-path/1/module-4.pt-br.md

```diff
@@ -0,0 +1,228 @@
+
+Atacantes podem hospedar seus próprios servidores DNS e rastrear solicitações de acesso. Sendo assim, as solicitações podem não ser “passivas”, e podem alertar atacantes sobre a investigação em andamento. Tome cuidado especialmente com hostnames que podem conter identificadores, como r2378r233yr39wjwr.example.com.
+
+## Pratique
+
+Escolha um domínio aleatório, certificando-se de que ele não esteja hospedado por uma proxy de distribuição de conteúdo/reversa como Cloudflare (é possível descobrir isso por meio de ferramentas como dig e a opção de NS ao buscar nameservers). Utilizando as ferramentas citadas, investigue o domínio, e tente explicar:
+
+- Onde o domínio está registrado e, caso a informação esteja disponível, quem o registrou?
+- Qual é o endereço IP do domínio?
+- Quem gerencia este endereço IP?
+- Onde se localiza o servidor?
+- (Caso haja acesso ao Shodan/Censys) Que serviços estão rodando nesse servidor?
+- Que outros domínios estão hospedados no mesmo IP?
+- Consegue encontrar subdomínios para esse domínio?
+
+## Checagem de habilidades
+
+Junto a pares ou a pessoa mentora mais experientes em investigação passiva de servidores na internet:
+
+- Complete o room (ou “sala”, em tradução adaptada) [do reconhecimento passivo (em inglês)](https://tryhackme.com/room/passiverecon) no TryHackMe.
+- Faça o exercício da seção “Pratique”, idealmente com um domínio diferente, e repasse o processo e descobertas com a pessoa par ou mentora. Peça para que revisem seu trabalho e comentem sobre o processo e os resultados obtidos. Pode ser um bom exercício discutir especificamente como encontrar subdomínios existentes no domínio, e sobre a precisão das pesquisas com geoIP relacionadas a estes domínios. Como atividade extra, fuce as configurações avançadas do dig e monte uma automação básica, por exemplo, solicitando uma lista de domínios a partir de um arquivo de texto, e forneça informações sobre eles.
+- Caso tenha uma mensagem real de phishing (ou, alternativamente, use um domínio de phishing no [PhishTank (em inglês; o nome vem de um trocadilho com “tanque de peixes”, ou “fish tank”)](https://phishtank.org/) e o analise — perceba que o site coleta domínios, e não mensagens. 
+
+## Recursos de aprendizagem
+
+{{% resource title="O que é uma URL?" languages="Chinês, Inglês, Francês, Japonês, Coreano, Russo, Espanhol" cost="Grátis" description="Resumo sobre o que são URLs, como são construídas, e que funcionalidades adicionais (âncoras, etc.) elas podem ter." url="https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL" %}}
+
+{{% resource title="Introdução ao DNS" languages="Vídeo em Inglês, texto em Árabe, Indonésio, Alemão, Espanhol, Francês, Italiano, Português, Vietnamita, Turco, Russo, Tailandês, Japonês, Coreano, Chinês, Taiwanês" cost="Grátis" description="Resumo básico sobre o funcionamento de DNS." url="https://aws.amazon.com/route53/what-is-dns/" %}}

```

Comment:
{{% resource title="Introdução ao DNS" languages="Vídeo em inglês, texto em árabe, indonésio, alemão, espanhol, francês, italiano, português, vietnamita, turco, russo, tailandês, japonês, coreano, chinês, taiwanês" cost="Grátis" description="Resumo básico sobre o funcionamento de DNS." url="https://aws.amazon.com/route53/what-is-dns/" %}}

Reply:
```suggestion
{{% resource title="Introdução ao DNS" languages="Vídeo em inglês, texto em árabe, indonésio, alemão, espanhol, francês, italiano, português, vietnamita, turco, russo, tailandês, japonês, coreano, chinês, taiwanês" cost="Grátis" description="Resumo básico sobre o funcionamento de DNS." url="https://aws.amazon.com/route53/what-is-dns/" %}}
```